### DISCUSSION TITLE
vendure + svelte

### DISCUSSION THREAD

**Discussion by MosheM2** [vendurehq/vendure#957] [Q&A]
Hey everyone,

after digging thru the docs of vendure and svelte I could'nt find a single way of how to integrate both vendure + svelte in my new web store  project. 

If somebody who tryed the same could point me into the right direction I'd be very thankfull!

Greetings Moe

**Comment by michaelbromley** [comment]
Hi Moe,

I'm not familiar with building Svelte apps, but a Vendure app is just like any other GraphQL-based client-server app. Is there a particular part you are having trouble with?

**Comment by maku** [comment]
As @michaelbromley said. You have to use **graphql** to access vendure (which is "standard" technology). I only tried some basics with sveltekit. But without problems.
